### ISSUE TITLE
Tag name alignment

### ISSUE THREAD

**Issue by JoelBender** [BrickSchema/Brick#118]
In Brick there is a `tag:Average` that is a `brick:Tag` with a label "Average".  In Haystack there is a `phScience:avg` that is a subclass of `ph:marker` with a label "avg".  Would it be safe to assert that the named individual `tag:Average` is also an instance of `phScience:avg` class?

In the **ph2rdf** ASHRAE 223 project the labels are language tagged:
```
phScience:current rdfs:label "current"@en,
        "courant"@fr,
        "电流"@zh ;
    ph:doc "Movement of fluid or electricity" ;
    ph:is ph:marker .
```
But this is not the current `defs.ttl` which is still version 3.9.7.

**Comment by gtfierro** [comment]
Yes, I think that's appropriate. If we were to do a ph/Brick alignment, should we work from the ph2rdf version? It looks to be more considered than the current `defs.ttl`

**Comment by JoelBender** [comment]
They are two different models, the `ph:is` is overloaded to mean subclass in one case:
```
phIoT:current-angle a owl:Class ;
    rdfs:subClassOf phScience:quantity ;
    ph:is phScience:quantity .
```
And something else in other cases:
```
phIoT:curStatus a owl:DatatypeProperty ;
    rdfs:range ph:str ;
    ph:is ph:str .
```
So the `defs.ttl` is built from some additional content that isn't in the TRIO files.  Because we are looking at 3.X content I would expect this to change substantially, maybe when the project is moved from bitbucket to GitHub.

In the new `defs.ttl` maybe `phScience:avg` is no longer a class but a named individual and `tag:Average` is transformed from an individual to a class of all instances of the `ph:entity` that "have the" `phScience:avg` tag.

**Comment by gtfierro** [comment]
Just making a note here that the `defs.ttl` file has been updated since we last looked, and we will need to produce a new tag alignment soon. Although we have Haystack to Brick inference in the `py-brickschema` package, we should probably produce an alignment of tags in the `alignments/` directory in this repo
